### PR TITLE
cdc-b2b changes

### PR DESCRIPTION
**Pull request by anjana-bl**

### CODE REVIEW

File: projects/storefrontapp/src/app/spartacus/features/organization/organization-administration-feature.module.ts [hunk 1]
```diff
@@ -14,10 +15,16 @@ import {
   AdministrationRootModule,
   ORGANIZATION_ADMINISTRATION_FEATURE,
 } from '@spartacus/organization/administration/root';
+import { environment } from '../../../../environments/environment';
 
+const extensions: Type<any>[] = [];
+
+if (environment.cdc) {
+  extensions.push(CdcAdministrationModule);
```
Comment (Platonn) [inline projects/storefrontapp/src/app/spartacus/features/organization/organization-administration-feature.module.ts]: CdcAdministrationModule should be lazy loaded, so it should be imported inside `AdministrationWrapperModule`. Not in the root injector

File: projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cdc/cdc.ts [hunk 2]
```diff
@@ -17,10 +17,10 @@ const user = getSampleUser();
 const nativeUser = getSampleUser();
 
 export const cdcB2BDelegateAdminUser = {
-  userId: '1002',
-  fullName: 'Anjana BL',
-  email: 'anjana.b.l+1@sap.com',
-  password: 'Anjana1201!',
+  userId: '80dfa58b-e24e-40a1-97c6-5e7f0c2bdd09',
+  fullName: 'Spartacus B2BAdmin',
+  email: 'spartacusb2b@mailinator.com',
```
Comment (Platonn) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cdc/cdc.ts]: Should we really send emails as part of our e2e to external mailboxes?  Did you consider other alternatives?

File: integration-libs/cdc/organization/administration/cdc-administration.module.ts [hunk 3]
```diff
@@ -5,51 +5,15 @@
...
-            {
-              provide: B2BUserService,
-              useExisting: CdcB2BUserService,
-            },
-            unitsCmsConfig.cmsComponents?.ManageUnitsListComponent?.providers ||
-              [],
-          ],
-        },
-      },
-    }),
```
Comment (Platonn) [inline integration-libs/cdc/organization/administration/cdc-administration.module.ts]: Do you mean this configuration is not needed, as I hoped?

Comment (anjana-bl) [reply integration-libs/cdc/organization/administration/cdc-administration.module.ts]: reverted this change

File: projects/storefrontapp/src/app/spartacus/features/organization/organization-administration-feature.module.ts (hunk 1 above)
Comment (anjana-bl) [reply projects/storefrontapp/src/app/spartacus/features/organization/organization-administration-feature.module.ts]: okay, reverting this change

**Review by Matejk00 — APPROVED** [review]

**Pull request merged**